doc/users/next_whats_new/selector_improvement.rst

```diff
@@ -0,0 +1,39 @@
+Selectors improvement: rotation, aspect ratio correction and add/remove state
+-----------------------------------------------------------------------------
+
+The `~matplotlib.widgets.RectangleSelector` and
+`~matplotlib.widgets.EllipseSelector` can now be rotated interactively.
+The rotation is enabled or disable by striking the *r* key
```

Comment:
```suggestion
The rotation is enabled or disabled by pressing the *r* key
```

Reply:
And that's also how the actions are labelled internally... but I don't think "strike" is typically used for keyboards (except perhaps the musical variety). Perhaps "typing" or "clicking", but I'd leave that to a native speaker to decide.